integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -69,26 +69,26 @@ def test_debug_trace_call_zero_eth_call(self):
         tx_hash = receipt["transactionHash"].hex()
 
         wait_condition(
-            lambda: self.tracer_api.send_rpc(method="eth_getTransactionByHash", params=[tx_hash])["result"] is not None,
-            timeout_sec=120,
+            lambda: self.web3_client.get_transaction_by_hash(tx_hash) is not None,
+            timeout_sec=10,
```

Comment:
Are you sure that 10 seconds is enough?

Reply:
I say we decrease them and see if the tests keep failing. It might not be enough, but so far the tests did not fail because of these reduced timeouts.